rotkehlchen/assets/utils.py

```diff
@@ -20,13 +20,12 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
-def add_ethereum_token_to_db(token_data: EvmToken) -> EvmToken:
+def add_ethereum_token_to_db(token_data: EvmToken, globaldb: 'GlobalDBHandler') -> EvmToken:
     """Adds an ethereum token to the DB and returns it
 
     May raise:
     - InputError if token already exists in the DB
     """
-    globaldb = GlobalDBHandler()
```

Comment:
why this change? Why in one function create it and in the other pass it? I think this is not a good change. I would revert this and keep the function's interface simple.